MovieAppPetProject/MovieAppPetProject/HomeScreen/NewTableViewCell.swift

```diff
@@ -26,17 +26,17 @@ class NewTableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionV
         layout.minimumLineSpacing = 10
         newCollectionView.collectionViewLayout = layout
     }
-    
+
     func configure(with movies: [NowPlayingResults]) {
         nowPlayingMovies = movies
         newCollectionView.reloadData()
     }
-    
+
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         print("tester: \(nowPlayingMovies.count)")
```

Comment:
print statements should be avoided to go through to dev 😄

Reply:
yes. removed.